src/components/link-preview/link-preview.pw.tsx

```diff
@@ -0,0 +1,266 @@
+        mount,
+        page,
+      }) => {
+        await mount(<LinkPreviewComponentTest as={as} />);
+
+        await expect(linkPreviewAs(page, as)).toBeVisible();
+      });
+    });
+
+    testData.forEach((title) => {
+      test(`should render Link Preview title prop using ${title} as special character`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<LinkPreviewComponentTest title={title} />);
+
+        await expect(page.getByText(title)).toBeAttached();
+      });
+    });
+
+    testData.forEach((description) => {
+      test(`should render Link Preview description prop using ${description} as special character`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<LinkPreviewComponentTest description={description} />);
+
+        await expect(page.getByText(description)).toBeAttached();
+      });
+    });
+
+    test("should render Link Preview with isLoading prop", async ({
+      mount,
+      page,
+    }) => {
+      await mount(<LinkPreviewComponentTest isLoading />);
+
+      const previewComponent = page.locator(`[data-component="preview"]`);
+
+      const count = await previewComponent.count();
+
+      expect(count).toBe(4);
+    });
+
+    test("should render Link Preview with Image props", async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <LinkPreviewComponentTest image={{ url: urlProp, alt: testCypress }} />
+      );
+
+      await expect(page.locator("img")).toHaveAttribute("src", urlProp);
+
+      await expect(page.locator("img")).toHaveAttribute("alt", testCypress);
+    });
+
+    test("should render Link Preview with url prop", async ({
+      mount,
+      page,
+    }) => {
+      await mount(<LinkPreviewComponentTest url="https://www.foo.com" />);
+
+      await expect(page.getByText("www.foo.com")).toBeAttached();
+    });
+  });
+
+  test.describe("check functionality of Link Preview component", () => {
+    test("should verify hover color of Link Preview component", async ({
+      mount,
+      page,
+    }) => {
+      await mount(<LinkPreviewComponentTest />);
+
+      const previewElement = linkPreview(page);
+      await previewElement.hover();
+
+      await expect(previewElement).toHaveCSS(
+        "background-color",
+        "rgb(204, 214, 219)"
+      );
+    });
+
+    test("should verify border outline color and width of Link Preview on focus", async ({
+      mount,
+      page,
+    }) => {
+      await mount<HooksConfig>(<LinkPreviewComponentTest />, {
+        hooksConfig: { focusRedesignOptOut: true },
+      });
+
+      await linkPreview(page).focus();
+
+      await checkGoldenOutline(linkPreview(page), "2px");
+    });
+
+    test("should verify border outline color of close icon on focus", async ({
+      mount,
+      page,
+    }) => {
+      await mount<HooksConfig>(
+        <LinkPreviewComponentTest
+          as="div"
+          // eslint-disable-next-line no-console
```

Comment:
do we need this line here?

Reply:
I think so, otherwise it returns warning for the next line because of the "console.log"